### ISSUE TITLE
[composer] error while loading shared libraries: libicuio.so.62

### ISSUE THREAD

**Issue by vincowl** [skycocker/chromebrew#4110]
Since PR #4103, I had troubles with curl dependencies that blocked `libxml2` upgrade for instance. That forced me to reinstall a fresh `chromebrew`. But now, I can't install `composer`. This is somewhat similar to #4104 except that neither can I install it from sources.

```
 crew install -s composer
composer: Dependency Manager for PHP
https://getcomposer.org/
version 1.10.6
Downloading source...
########################################################################################################################################################################################### 100.0%    -=O=-           #         #           #            #                                                                                                                                         
Archive downloaded
Unpacking archive using 'tar', this may take a while...
gcc (GCC) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
Preconfiguring package...
php: error while loading shared libraries: libicuio.so.62: cannot open shared object file: No such file or directory
composer failed to install: `php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"` exited with 127
```

**Comment by vincowl** [comment]
OK, I solved this. I needed to recompile from sources php, then install composer. This is exactly the scope of #4104.

To solve other packages installation errors since I installed `curl`, I had to recompile `curl` from sources since it depends on `libmetalink` which has been regenerated using the new `icu4c` version. So, @uberhacker @cstrouse, **we must regenerate `curl` binaries asap** otherwise people having curl installed and an updated chromebrew won't be able to install anything.
